Superlib.js

```diff
@@ -0,0 +1,633 @@
+		
+				let index = content.indexOf('】');
+				if (index <= -1) continue;
+		
+				content = trim(content);
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【作 者】":
+						doPerson(item, value);
+						break;
+					case "【会议名称】":
+						item.conferenceName = value;
+						break;
+					case "【会议录名称】":
+						item.series = value;
+						break;
+					case "【日 期】":
+						item.date = value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value.replace(/隐藏更多/g, '');
+						break;
+					case "【关键词】":
+						doTag(item, value);
+						break;
+					case "【作者联系方式】":
+						item.place = value;
+						break;
+					default:
+						break;
+				}
+			}
+			break;
+		case '专利':
+			item.title = ZU.trimInternal(text(doc, 'h1.title'));
+			infos = doc.querySelectorAll('.content li');
+			for (var section4 of infos) {
+				let content = ZU.trimInternal(section4.textContent);
+				if (!content || content.length <= 0) continue;
+		
+				let index = content.indexOf('】');
+				if (index <= -1) continue;
+		
+				content = trim(content);
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【申请号】":
+						item.applicationNumber = value;
+						break;
+					case "【申请人】":
+						doInventor(item, value);
+						break;
+					case "【发明人】":
+						doInventor(item, value);
+						break;
+					case "【申请日期】":
+						item.filingDate = value;
+						break;
+					case "【地 址】":
+						item.place = value;
+						break;
+					case "【专利类型】":
+						item.extra = '专利类型: ' + value;
+						break;
+					case "【IPC号】":
+						item.extra += '; IPC号: ' + value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value;
+						break;
+					default:
+						break;
+				}
+			}
+			break;
+	}
+	
+	if (item.title.startsWith('《') && item.title.endsWith('》')) {
+		item.title = item.title.replace(/《|》/g, '');
+	}
+	
+	// 如果抓取失败，请配合油猴脚本使用：https://greasyfork.org/zh-CN/scripts/408790
+	if (pdfurl) {
+		item.attachments.push({
+			url: pdfurl,
+			title: 'Full Text PDF',
+			mimeType: 'application/pdf'
+		});
+	} else {
+		var download = doc.querySelector('.link a');
+		if (download && download.textContent == 'PDF下载') {
+			item.attachments.push({
+				url: download.href,
+				title: 'Full Text PDF',
+				mimeType: 'application/pdf'
+			});
+		}
+	}
+	
+	item.complete();
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://book.ucdrs.superlib.net/views/specific/2929/bookDetail.jsp?dxNumber=000018108791&d=F788C1F57DDBFD163B5EDCB5371550F0&fenlei=02140804",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "认知跃迁",
+				"creators": [
+					{
+						"lastName": "（日）本田晃一",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "朱运程",
+						"creatorType": "translator",
+						"fieldMode": 1
+					}
+				],
+				"date": "2019.04",
```

Comment:
Didn't want to note this everywhere you set the date, but these should be normalized with `ZU.strToISO`.